src/components/Canvas/ImagesBlock.vue

```diff
@@ -1,85 +1,92 @@
 <template>
-  <div v-for="(relateImage, index) in relatedImages" :key="index">
+  <div v-for="(relatedImage, index) in imageBlock.relatedImages" :key="index">
     <image-element
+      v-if="relatedImage"
       :ref="'image-element-' + index"
       :imagePosition="calculatePositions(index)"
       :isTop="true"
       :isLeft="false"
-      :focus="false"
-      :tag="relateImage.tag"
-      :imageId="relateImage.imageId"
+      :focus="relatedImage.hasFocus"
+      :tag="relatedImage.tag"
+      :imageId="relatedImage.imageId"
       :imageFactor="imageFactor"
+      :class="{'last_block': isFull && currentGlobalPosition === 0 && relatedImage.imageId !== imageBlock.oldCentralImage && !relatedImage.hasFocus}"
```

Comment:
do we need the isFull?

Reply:
yes, we need isFull but the name is not good and will change in the next push